letsencrypt/auth_handler.py

```diff
@@ -113,6 +115,22 @@ def _solve_challenges(self):
                     cont_resp = self.cont_auth.perform(self.cont_c)
                 if self.dv_c:
                     dv_resp = self.dv_auth.perform(self.dv_c)
+                    for achall, response in zip(self.dv_c, dv_resp):
+                        simple_verify_return = True
+                        if isinstance(response, challenges.HTTP01Response):
+                            simple_verify_return = response.simple_verify(
+                                    achall.chall, achall.domain,
+                                    achall.account_key.public_key(),
+                                    self.config.http01_port)
+                        elif isinstance(response, challenges.TLSSNI01Response):
+                            response.simple_verify(
+                                    achall.chall, achall.domain,
+                                    achall.account_key.public_key())
+                        if not simple_verify_return:
+                            logger.warning("Self-verify of %s "
+                                    "challenge for domain %s failed.",
```

Comment:
- indentation is broken
- can we please refer to the `achall.uri` in the log message? there might be multiple challenges of the same type...